### ISSUE TITLE
Error Changing Folder Name of Markdown Page

### ISSUE THREAD

**Issue by thekenshow** [newbthenewbd/grav-plugin-tinymce-editor#31]
I've run into a new error when trying to change a page folder name. In this case, it's a Markdown page (i.e., TinyMCE is not enabled on this custom blueprint). The error:

```
 Whoops \ Exception \ ErrorException (E_WARNING)
DOMDocument::loadHTML(): Tag section invalid in Entity, line: 1

/user/plugins/tinymce-editor/tinymce-editor.php:17 
```

The page contains:

```
<section class="offs-xxl">
	<div class="container">
    	<div class="item-content">
        	<div class="lead">Genuine Mackie replacement parts.</div>
			<p>[ti file=Thump-Series-MACKIE-THUMP-SERIES.csv class="table table-hover"]</p>
		</div>
   	</div>
</section>
```

**Comment by newbthenewbd** [comment]
And to think I felt so clever when I disabled the libxml warning suppression, finding no adverse effects within the few minutes of testing... Fixed in v1.2.5, hopefully! :upside_down_face:

Thanks for reporting!

**Comment by thekenshow** [comment]
Tested and it works fine - thanks!
